python_ta/typecheck/base.py

```diff
@@ -76,6 +76,54 @@ def create_Callable(args: Iterable[type], rtype, poly_vars=None):
 }
 
 
+def binary_op_hints(op, args):
+    """Return an appropriate 'hint' or suggestion given the binary operation and operand types."""
+    if op == '+':
+        if int in args and str in args:
+            return "Perhaps you wanted to cast the integer into a string or vice versa?"
+
+
+def op_to_name_binary(op):
+    """Return the name of the method corresponding to binary op."""
+    if op == '+':
```

Comment:
Use a dictionary to store the mapping rather than writing them all out like this.